### PR TITLE
Add how to deploy etherpad tutorial

### PR DESCRIPTION
**Pull request by squidboylan**

### CODE REVIEW

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+How to deploy Etherpad on DreamCompute
+======================================
+
+Etherpad is a web application that lets you collaborate with others in a text
+editor, much like an open source Google Docs alternative.
+
+Setting up a server
+~~~~~~~~~~~~~~~~~~~
+
+The first step to deploying Etherpad is to launch a server to run it on, this
```
Comment (cjdh) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Reword:
The first step to deploying Etherpad is to launch a server to run it on. For example in this tutorial, Ubuntu Xenial server is used. View the following link for documentation on how to do this.
* ADD LINK HERE 
You also need to expose port 8080 to incoming traffic, as that is blocked by default. View the following link for documentation on how to add security rules.
* ADD LINK HERE

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+Once you have your server up and running the next step is to install all of
+Etherpad's dependencies:
+
+.. code:: bash
+
+    apt-get install gzip git curl python libssl-dev pkg-config build-essential
+    apt-get install nodejs npm
+
+Next we have to symlink /usr/bin/nodejs to /usr/bin/node because Etherpad will
```
Comment (cjdh) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Reword to remove 'we':
Next you must symlink /usr/bin/nodejs to /usr/bin/node because Etherpad will try to use that path. 

Reword and fix misspelling:
Most Linux distributions install nodejs in /usr/bin/node. This step is only necessary on Ubuntu servers since it doesn't install nodejs because of another package.

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+    git clone git://github.com/ether/etherpad-lite.git
+
+Configuration
+-------------
+
+Networking
+^^^^^^^^^^
+
+Now comes the configuration of Etherpad. By default it will run on port 9001,
```
Comment (cjdh) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Reword for present tense and grammar:
Now comes the configuration of Etherpad. By default it runs on port 9001. Change it to run on port 8080 by editing the settings.json file:

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+.. code:: bash
+
+    "port" : 8080,
+
+Database
+^^^^^^^^
+
+By default Etherpad uses dirtyDB to store its data, but it's recommended you
+use something else in a production environment and only use dirtyDB for
+testing. We are going to use MySQL to store our data, but Etherpad supports
```
Comment (cjdh) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Reword to remove 'we' and 'our':
This tutorial uses MySQL to store data, but Etherpad supports other databases such as PostgreSQL and SQLite.

If you don't have MySQL running, follow this tutorial on how to install it. Once you have that running, connect to MySQL and create a database for Etherpad to use:

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    "dbType" : "mysql",
+    "dbSettings" : {
+                     "user"    : "etherpad",
+                     "host"    : "localhost",
+                     "password": "ETHERPAD USER PASSWORD",
+                     "database": "etherpad",
+                     "charset" : "utf8mb4"
+                   },
+
+Your configuration may be a bit different depending on how you have MySQL
```
Comment (cjdh) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Split setup into two words (or change it to "configured")

Your configuration may be a bit different depending on how you have MySQL set up, adjust the values accordingly.

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+setup, adjust the values accordingly.
+
+Starting Etherpad
+~~~~~~~~~~~~~~~~~
+
+Now that everything is configured we can finally start Etherpad. I recommend
```
Comment (cjdh) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Reword to remove 'we':
Now that everything is configured, you can finally start Etherpad.

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+running it inside screen or tmux so that it continues to run after you
+disconnect from the server.
+
+.. code:: bash
+
+    tmux
+    bin/run.sh
+
+Now Etherpad is running, confirm that it works by going to http://IP:8080 ,
```
Comment (cjdh) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Adjust grammar:
Etherpad is now running. Confirm it works by going to http://IP:8080. Make sure to replace "IP" with the IP address of your server.

**Comment by cjdh** [comment]
Made several adjustments to grammar, misspellings and verbage.

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst (hunk 1 above)
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Is that the first step always? What if the user already has a server?

I'd reword with the instructions to launch a server as an option.  Like:

`This tutorial describes how to launch an Ubuntu 16.04LTS (Xenial Xwhatever its called), install and configure all the packages necessary for Etherpad.`

I like the suggestion by @cjdh to add links, but I'd add them below when the server is up and running and ready to be tested.

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+Configuration
+-------------
+
+Networking
+^^^^^^^^^^
+
+Now comes the configuration of Etherpad. By default it will run on port 9001,
+change it to run on port 8080, by editing the settings.json file:
+
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Here I'd add the note with a link to how to open port 8080, to change the security groups.

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst (hunk 4 above)
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: +1 to @cjdh  only I'd use `MySQL/MariaDB` since they're similar and we have documentation for both.

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+use something else in a production environment and only use dirtyDB for
+testing. We are going to use MySQL to store our data, but Etherpad supports
+other databases such as PostgreSQL and SQLite.
+
+If you don't have MySQL running, follow `our tutorial on how to install it
+<215879487>`__. Once you have that running, connect to MySQL and create a
+database for Etherpad to use:
+
+.. code:: bash
+
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: in this snippet I'd also put the `mysql` client command, while you're at it, not just the SQL command

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+Your configuration may be a bit different depending on how you have MySQL
+setup, adjust the values accordingly.
+
+Starting Etherpad
+~~~~~~~~~~~~~~~~~
+
+Now that everything is configured we can finally start Etherpad. I recommend
+running it inside screen or tmux so that it continues to run after you
+disconnect from the server.
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: really? there is no systemd nor init script?

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst (hunk 8 above)
Comment (squidboylan) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: even though the article for that is linked above?

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst (hunk 10 above)
Comment (squidboylan) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: no because it's not packaged in apt, we're just cloning the repo

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 11]
```diff
@@ -0,0 +1,169 @@
...
+Etherpad is a web application that lets you collaborate with others in a text
+editor, much like an open source Google Docs alternative.
+
+Setting up a server
+~~~~~~~~~~~~~~~~~~~
+
+The first step to deploying Etherpad is to launch a server to run it on. For
+example in this tutorial, an Ubuntu Xenial server is used. View the following
+link for documentation on how to do this.
+https://help.dreamhost.com/hc/en-us/articles/215912848-How-to-launch-and-manage-servers-with-the-DreamCompute-dashboard.
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Links should be named properly. For this use:

Read `how to launch servers using DreamCompute dashboard <https://help.dreamhost.com/hc/en-us/articles/215912848-How-to-launch-and-manage-servers-with-the-DreamCompute-dashboard>`_

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+You also need to expose port 8080 to incoming traffic,
+as that is blocked by default. View the following link for documentation on how
+to add security rules.
+https://help.dreamhost.com/hc/en-us/articles/215912838-How-to-configure-access-and-security-for-DreamCompute-instances
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: As with the link above:

Read `how to configure access and security for DreamCompute instances <URL>`_ etc.

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 13]
```diff
@@ -0,0 +1,169 @@
...
+should be changed to:
+
+.. code:: bash
+
+    "port" : 8080,
+
+.. Note::
+
+    See the following link for how to expose port 8080 to incoming traffic.
+    https://help.dreamhost.com/hc/en-us/articles/215912838-How-to-configure-access-and-security-for-DreamCompute-instances
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: Same as above for the links: make the text more readable

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 14]
```diff
@@ -0,0 +1,169 @@
...
+
+Database
+^^^^^^^^
+
+By default Etherpad uses dirtyDB to store its data, but it's recommended you
+use something else in a production environment and only use dirtyDB for
+testing. This tutorial uses MySQL to store data, but Etherpad supports other
+databases such as PostgreSQL and SQLite.
+
+If you don't have MySQL running, follow `this <215879487>`__. Once you have
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: `this` is what exactly? Make that anchor link more explicit about what the link points to

File: source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+                   },
+
+Your configuration may be a bit different depending on how you have MySQL
+configured, adjust the values accordingly.
+
+Creating a systemd service
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The best way to run Etherpad is to create a systemd service for it and create a
+user for it to run as. To create a systemd service copy the following into
```
Comment (smaffulli) [inline source/dreamcompute/tutorials/deploy-etherpad-on-dreamcompute.rst]: ``to create ... To create`` feels like a repetition. Remove the second one and go straight to instructions:

user for it to run as. Create the file `/etc/systemd/service/etherpad-lite.service` and copy the text below in it:
